es-vue-base/src/lib-utils/nav-bar-global-content.js

```diff
@@ -72,9 +72,7 @@ export default (
                         },
                         {
                             name: 'How much do heat pumps cost?',
-                            link: `${
-                                ES_DOMAIN
-                            }/clean-heating-cooling/air-source-heat-pumps/costs-and-benefits-air-source-heat-pumps/`,
+                            link: `${ES_DOMAIN}/clean-heating-cooling/air-source-heat-pumps/costs-and-benefits-air-source-heat-pumps/`,
```

Comment:
From the github action/es-lint warning to add '// eslint-disable-next-line max-len' above this line